Core/clim-core/menu-choose.lisp

```diff
@@ -215,8 +215,13 @@
           (when deexpose ; Checkme as well.
             (disown-frame fm frame)))))))
 
+;; The MENU-ITEM object is described in the specification of the
+;; function FRAME-MANAGER-MENU-CHOOSE. -- jd 2020-07-02
 (define-presentation-type menu-item ())
 
+(define-presentation-method presentation-typep (object (type menu-item))
+  (presentation-translator-menu-item-p (alexandria:ensure-car object)))
+
```

Comment:
@dkochmanski This method might be needed for items in "menu-choose" menus. Testing with "Accepting-values" test, nothing in the menu-choose menu is highlight-able.

Reply:
After reading the spec again it seems that it is not specified anywhere what is the display object. I'm fixing this predicate to always return T. Thanks.